server/lib/orcasite/notifications/resources/notification.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Orcasite.Notifications.Notification do
+  use Ash.Resource,
+    extensions: [AshAdmin.Resource],
+    data_layer: AshPostgres.DataLayer,
+    notifiers: [Orcasite.Notifications.Notifiers.NotifySubscriptions]
+
+  alias Orcasite.Notifications.{Event, SubscriptionNotification, Subscription}
+
+  resource do
+    description """
+    A notification object. Once created, this will be sent to any number of subscribers (api clients,
+    newsletters, individuals, admins, etc) through their subscribed channels for the given event type
+    (confirmed candidate created, detection verified, etc).
+    """
```

Comment:
I find the distinction here in these descriptions between `notification` and `subscription_notification` slightly confusing. How I understand it (mostly based on previous discussions): there's one `notification` object per event, and then a `subscription_notification` is created for each subscriber. So `subscription_notification` represents the actual individual notification instance that goes out to a specific subscriber, while `notification` is like a blueprint / template. Do I have that right?

I think it's inherently confusing so I want to make sure we document it properly

Reply:
Yep that's exactly it. `SubscriptionNotification` is the join table between a `Notification` (send this event to everyone subscribed) and `Subscription` (subscribe to an event type with a particular channel)